src/gengowatcher/watcher.py

```
@@ -1200,13 +1932,38 @@ def _run_websocket_monitor(self):
+            if debug_url:
+                changed = await asyncio.to_thread(
+                    self._sync_session_from_browser,
+                    fail_hard=self.config.getboolean(
+                        "WebSocket", "session_sync_fail_hard", fallback=True
+                    ),
+                    alert_on_failure=self.config.getboolean(
+                        "WebSocket",
+                        "session_sync_alert_on_failure",
+                        fallback=True,
+                    ),
+                )
```

Ruff (0.15.5)

[error] `await` should be used within an async function

(PLE1142)